labs/lab-03/tasks/goto-algs/vector_max/tests/test_vector_max.c

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+
+#include "graded_test.h"
+
+int vector_max(int *v, int len);
+
+static int test_1(void)
```

Comment:
```suggestion
static int test_max_middle_of_array(void)
```
Same thing: update the checker output in `README.md`.